openlibrary/macros/EditButtons.html

```diff
@@ -1,4 +1,4 @@
-$def with (comment=None)
+$def with (comment=None, submit_markup='')
```

Comment:
This macro is referenced in a BUNCH of other places ; we can't modify it like this -- you know unless we _do_ want to delete the Save button on every page but the book page :P It's a good idea to check for usages of a template/function whenever modifying its arguments. I'm going to roll back the last commit :+1: